### ISSUE TITLE
Parcels.service

### ISSUE THREAD

**Issue by SaintAngeLs** [SaintAngeLs/courier_app#30]

**Comment by SaintAngeLs** [comment]
@an2508374, Would you need some help?

**Comment by eggwhat** [comment]
There are missing events for external events in `SwiftParcel.Services.Parcels.Application/Events`. When new customer is created it won't create its instance in parcels service (because there are no events handling such events).

**Comment by eggwhat** [comment]
One small question. If I am not mistaken, address may not have apartment number (just building number). So we should maybe change that. 
```csharp
CheckAddressElement("apartment number", apartmentNumber);
            address.ApartmentNumber = apartmentNumber;
```

**Comment by SaintAngeLs** [comment]
@an2508374, Please, check the service one more time.

**Comment by SaintAngeLs** [comment]
- GetParcels for the client side and the office worker

**Comment by SaintAngeLs** [comment]
This issue was closed, and opened as the #71.
